### DISCUSSION TITLE
What are the consequences of disabling 2-factor authentication for an account?

### DISCUSSION THREAD

**Discussion by WilliamBonvini** [community/community#64220] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi there, 
I have requested GitHub to disable two-factor authentication for one of my accounts (not the one from which I'm currently writing this post). If you're interested in learning how I made this request, you can find the details in the concluding part of this post. The outcome of my request is reflected in the following email:

<img width="594" alt="Screenshot 2023-08-18 at 17 12 29" src="https://github.com/community/community/assets/37834150/454a76c0-df10-47b2-9a1f-f751fbf93904">

Now, I have a question:

What potential consequences would arise from removing my account from the organization I am employed by? Would the commit history and the pull requests that I have generated—both those that are open and those that are closed—be disrupted in any way? My assumption is that they would remain intact. However, I am curious about how these elements could be attributed to an account that is no longer affiliated with the organization.

My ultimate objective is to reclaim access to my account, regain entry to my organization, and resume my work on the commits and pull requests that I had been actively contributing to.

Thank you in advance for your help,  I attempted to reach out to the official support, but it appears that there is no available method to do so.

### [additional info, not needed to answer the question] Instructions for Requesting 2-Factor Authentication Deactivation

Try to log in, then follow the red circles! 

<img width="300" alt="Screenshot 2023-08-18 at 17 02 21" src="https://github.com/community/community/assets/37834150/50e8385d-b517-401b-b363-04bf216e9c8e">  

\\

<img width="400" alt="Screenshot 2023-08-18 at 17 02 41" src="https://github.com/community/community/assets/37834150/66d54a4e-bd7d-4e69-99ac-c6db3a195565">  

\\

<img width="400" alt="Screenshot 2023-08-18 at 17 03 20" src="https://github.com/community/community/assets/37834150/5c53d272-afcc-49d3-8d85-f38a8e7f0f0e">   

\\

<img width="400" alt="Screenshot 2023-08-18 at 17 05 13" src="https://github.com/community/community/assets/37834150/9bd496bc-4305-4023-9333-84d1b94fb9e4">  

\\

<img width="400" alt="Screenshot 2023-08-18 at 17 05 37" src="https://github.com/community/community/assets/37834150/65add9df-1dad-4a63-9dac-f466b1f2751e">  

\\

<img width="400" alt="Screenshot 2023-08-18 at 17 06 04" src="https://github.com/community/community/assets/37834150/2b72ab3d-5755-4706-9d1a-2d8e35a1bc84">

**Comment by advayc** [comment, marked as answer]
Removing your GitHub account from your organization could result in losing access to organization-specific content, while your commit history and pull requests would remain but not directly associated with your account in that context. To regain access and rejoin the organization, consider reaching out to GitHub Support, communicating with organization admins, and following any available access restoration processes.

**Reply by WilliamBonvini** [reply]
Thank you ;) I simply care that my commit history and pull request would remain in the system. 
Unfortunately, I don't know how to contact GitHub support directly since the website always suggests I write my doubts in this community forum. Is there a way to contact them easily? I don't have an enterprise account.

**Comment by WilliamBonvini** [comment]
In the end I disabled 2FA authentication,  was removed from my organization, and re-entered my organization, but nothing got lost in the process. Still had all my PRs as I thought. 
Thanks!
